lib/support/requests/accounts_permissions_and_training_request.rb

```diff
@@ -15,7 +15,7 @@ class CreateOrChangeUserRequest < Request
         end
       end
       validates_presence_of :action, :requested_user
-      validates :action, inclusion: { in: ["create_new_user", "change_user"] }
+      validates :action, inclusion: { in: ["create_new_user", "change_user", "unsuspend_user"] }
```

Comment:
I added `unsuspend_user` too because it was requested as a "pretty please" on the trello card.